lgc/builder/ImageBuilder.cpp

```diff
@@ -705,14 +705,15 @@ Value *ImageBuilder::CreateImageSample(Type *resultTy, unsigned dim, unsigned fl
       Value *addr = load->getOperand(0);
       unsigned byteOffset = 0;
       for (;;) {
-        if (auto bitcast = dyn_cast<BitCastInst>(addr)) {
+        if (auto bitcast = dyn_cast<BitCastOperator>(addr)) {
```

Comment:
Why we change Inst class to Operator?

Reply:
That also covers the case that addr is a ConstantExpr whose operator is bitcast.

Reply:
I see. Better to mention this as a comment because most time we just make a instruction cast.

Reply:
OK. (Same for GEPOperator below.)